src/main/java/net/minecraftforge/client/model/geometry/IUnbakedGeometry.java

```diff
@@ -28,6 +30,13 @@
 {
     BakedModel bake(IGeometryBakingContext context, ModelBaker baker, Function<Material, TextureAtlasSprite> spriteGetter, ModelState modelState, ItemOverrides overrides, ResourceLocation modelLocation);
 
+    /**
+     * Resolve parents of nested {@link BlockModel}s which are later used in
+     * {@link IUnbakedGeometry#bake(IGeometryBakingContext, ModelBaker, Function, ModelState, ItemOverrides, ResourceLocation)}
+     * via {@link BlockModel#resolveParents(Function)}
+     */
+    void resolveParents(Function<ResourceLocation, UnbakedModel> modelGetter);
```

Comment:
Make this a default NO-OP method.
Since this is what is the most used implementation.

Reply:
In Forge itself, yes, but in most mods it'll very likely be the other way around.
My main concern is the discoverability of this for existing implementations since this issue is anything but obvious. If you deem this an acceptable risk, I'll change it to a default.